main.nf

```diff
@@ -1236,28 +1244,36 @@ if (params.binning_map_mode == 'all'){
 
 process bowtie2 {
     tag "$assembler-$name"
+    publishDir "${params.outdir}/Assembly/${assembler}/${name}_QC", mode: params.publish_dir_mode,
+        saveAs: {filename -> filename.indexOf(".bowtie2-log") > 0 ? filename : null}
 
     input:
     set val(assembler), val(name), val(group), file(assembly), val(sampleToMap), val(sampleGroup), file(reads) from ch_bowtie2_input
 
     output:
     set val(assembler), val(name), file("${assembler}-${name}-${sampleToMap}.bam"), file("${assembler}-${name}-${sampleToMap}.bam.bai") into ch_assembly_mapping_for_metabat
+    set val(assembler), val(name), val(sampleToMap), file("*.bowtie2-log") into ch_assembly_mapping_stats
 
     when:
     !params.skip_binning
 
     script:
-    def name = "${assembler}-${name}-${sampleToMap}"
+    def NAME = "${assembler}-${name}-${sampleToMap}"
```

Comment:
good point, that was not nice! But I would suggest using a variable name in lower case, e.g. `map_name`, otherwise it seems as if there is another reason why is is suddenly in upper case.

Reply:
done